### ISSUE TITLE
It would be more useful to import the compiled template than the generated text

### ISSUE THREAD

**Issue by bary12** [thoughtbyte/parcel-plugin-ejs#1]
ejs templates are meant to be used with different render data, and when importing them, there is no way to pass render data.
I think it will be more useful to import the compiled template instead.

## How it works currently
```ejs
<button><% buttonText %></button>
```

```js
import rendered from './template.ejs';
```

results in error: buttonText is not defined

## The Suggestion

With the same ejs file,

```js
import template from './template.ejs';
const rendered = template({
    buttonText: 'Click Me'
});
```

This change would be a matter of changing [this line](https://github.com/thoughtbyte/parcel-plugin-ejs/blob/master/src/EJSAsset.js#L31) from `return compiled();` to `return compiled;`.

**Comment by wulisensen** [comment]
I very much agree with your suggestion.

I'm trying to use parcel from webpack.Now, i get a trouble about parcel-plugin-ejs:  names is not defined.

`
<% if (names && names.length) { %>
	<% names.forEach(function(src,index){ %><video class="advideo" data-tag="<%= index %>"style="position:absolute;top:0;left:0;display:none;" poster="http://p1.ifengimg.com/a/2016/1021/loading0.jpg" src="<%= src %>" id="adPlayer" preload="auto"></video><% }) %>
<% } %>
`

How to solve this problem now...

**Comment by bary12** [comment]
@thoughtbyte any thoughts about this? Take in mind that this is how webpack's [ejs-loader](https://github.com/okonet/ejs-loader) works, and it seems to be a much better use for this plugin.

**Comment by thoughtbyte** [comment]
fixed in 0.2.0

**Comment by LucienLee** [comment]
How about work as pug in parcel?
put the data into ejs.config.js and compile with it
https://github.com/parcel-bundler/parcel/blob/75310e15f6fa3c8a0ce6593ada1bcca00240ea54/packages/core/parcel-bundler/src/assets/PugAsset.js#L36
